### PR TITLE
chore(accessibility): Increase the contrast of modal close button to 3:1

### PR DESCRIPTION
**Pull request by patrickpaul**
Increases the contrast ratio of the icon to >3:1 which is the WCAG AA standard for non-text UI.

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @patrickpaul has signed the CLA. Thanks for the pull request!

File: src/components/modal/ModalDialog.js [hunk 1]
```diff
@@ -72,7 +72,7 @@ class ModalDialog extends React.Component<Props> {
                 className="modal-close-button"
                 onClick={this.onCloseButtonClick}
             >
-                <IconClose color="#999EA4" height={18} width={18} />
+                <IconClose color="#909090" height={18} width={18} />
```
Comment (patrickpaul) [inline src/components/modal/ModalDialog.js]: Color choice of #909090 has been approved by DG in Box Design.

**Pull request merged**